linode/manager
# fix: [M3-6454] - Show error for PayPal payments

## Description 📝
Display an error Notice to the user when PayPal fails to launch after clicking the `PayPalChip` in the `AddPaymentMethodDrawer` or the `PayPalButton` in the `PaymentDrawer`. Previously a pop-up window would flash and the user would be unaware that an error had occurred.

Also, if the `BrainTreePayPalButtons` fail to load initially, an error `ToolTipIcon` is shown to the user.

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screen Shot 2023-04-27 at 9 11 05 AM](https://user-images.githubusercontent.com/119514965/234924045-e6df66b6-6608-4df6-ba05-790622ce300b.jpg) | ![Screen Shot 2023-04-27 at 8 30 46 AM](https://user-images.githubusercontent.com/119514965/234919817-d7cd63b8-b1da-4fc1-bb32-7b36b7d5819f.jpg) |
| ![Screen Shot 2023-04-27 at 9 06 38 AM](https://user-images.githubusercontent.com/119514965/234922279-d7191b6b-6594-41c6-a848-0a9df4001736.jpg) | ![Screen Shot 2023-04-27 at 8 30 25 AM](https://user-images.githubusercontent.com/119514965/234920021-b173d6ad-0f3e-4959-bde1-8c29b1e3415e.jpg) |
| ![Screen Shot 2023-04-27 at 9 05 00 AM](https://user-images.githubusercontent.com/119514965/234921108-5688ddf2-e1d2-4cd8-8cb0-282310bb6b57.jpg) | ![Screen Shot 2023-04-27 at 8 36 56 AM](https://user-images.githubusercontent.com/119514965/234919405-9812784e-b6de-4bfd-b658-cd1272969d56.jpg) |

## How to test 🧪
1. Navigate to the Billing page on your local environment and click on either the `Make A Payment` or `Add Payment Method` buttons.
2. Click on the PayPal option.
3. You should see a PayPal modal flash and then the error Notice should be displayed in the appropriate payment drawer.
4. The `BrainTreePayPalButtons` sometimes fail to load which would result in an empty space, try reloading the page a few times and if the failure occurs, the error `TooltipIcon` should appear in their place.
